waku/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -233,7 +239,7 @@ proc getInt(s: PostgresDriver,
             Future[ArchiveDriverResult[int64]] {.async.} =
   # Performs a query that is expected to return a single numeric value (int64)
 
-  let rowsRes = await s.connPool.query(query)
+  let rowsRes = await s.readConnPool.query(query, newSeq[string](0))
```

Comment:
Why is it necessary to add here `newSeq[string](0)`?

Reply:
Good catch! Not needed indeed :)
I'll change it now